app/controllers/frequently_asked_questions_controller.rb

```diff
@@ -1,44 +0,0 @@
-class FrequentlyAskedQuestionsController < ApplicationController
-  load_and_authorize_resource
-
-  def index
-    @title = 'FAQ'
-    @meta_description = "Answers to some commonly asked questions when dealing with a custom software company and dealing with BiteSite specifically."
-    @frequently_asked_questions = @frequently_asked_questions.in_order
-  end
-
-  def show
-  end
-
-  def new
-  end
```

Comment:
@NRaeewal I think it's better to transfer all CRUD functionality to a new controller and re-use existing rails forms to create/update/delete questions. I added a method to check a user role (window.is...), so you can show links for adding new questions or updating/deleting somewhere in UI to keep the ability to add questions from UI.